packages/dashboard/src/components/SubwayNav/SubwayNavPage.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
+import {
+  SubwayNavBasicExample,
+  SubwayNavWizardCompleteExample,
+  SubwayNavBasicDifferentStatesExample,
+  SubwayNavDisabledStepsExample,
+  SubwayNavSubStepsExample
+} from '@uifabric/dashboard/lib/components/SubwayNav/examples';
```

Comment:
This looks to be the line that's breaking the build. I see you have an index.ts file there that exports each of these examples, so I'm not sure why it's complaining.

```
ERROR: 1 import path(s) do not reference physical files. This can break AMD imports. Please ensure the following imports reference physical files:
-------------------------------------
  components/SubwayNav/SubwayNavPage.tsx: @uifabric/dashboard/lib/components/SubwayNav/examples
```

Reply:
Thank you.  I fixed it and got a clean build.